jakazzy/Trello-clone
# add tailwind css

#### Pre-Submission Checklist






- [ ] Your pull request targets the `master` branch of Kudobuzz.
- [ ] All new and existing tests pass the command `npm test`. Use `git commit --amend` to amend any fixes.

#### Checklist:




- [ ] All test are passing.
- [x] Tested changes locally using:
      i.Browser
      ii. incognito window
      iii. guest window
- [ ] Current changes in the pr have the necessary test.
- [x] Your commit messages format follows our commit message guideline.

#### What does this pr do?
This PR adds cards and column to project

#### How should this be manually tested?
git clone and git checkout branch 'create a trello board'
Check package.json to see tailwind installed with its configuration in the project



##### Any background context you want to provide?
None



#### What are the relevant issues?

#5
